db/database.go

```diff
@@ -706,6 +707,20 @@ func (context *DatabaseContext) Authenticator() *auth.Authenticator {
 	return authenticator
 }
 
+func (context *DatabaseContext) UserXattrsEnabled() bool {
```

Comment:
Unused, but I guess may be needed for future PRs?

Reply:
Yeah that was the plan... I'll remove for now though. Empty string check on the other helper func is sufficient for now anyways.